packages/terra-slider/src/Slider.module.scss

```diff
@@ -0,0 +1,212 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './orion-fusion-theme/Slider.module';
+@import './clinical-lowlight-theme/Slider.module';
+
+:local {
+  .slider {
+    align-items: center;
+    display: inline-flex;
+    min-height: 44px;
+  }
+
+  .label {
+    color: var(--terra-slider-label-color, #1c1f21);
+    font-weight: 700;
+    margin-right: 7px;
+    text-align: center;
+    white-space: initial;
+    width: 20%;
+  }
+
+  .min-label {
+    color: #6f7477;
+    font-size: var(--terra-slider-range-label-font-size, 12px);
+    font-weight: 400;
+    margin-right: 7px;
+    text-align: left;
+    white-space: initial;
+  }
+
+  .max-label {
+    color: #6f7477;
+    font-size: var(--terra-slider-range-label-font-size, 12px);
+    font-weight: 400;
+    margin-left: 7px;
+    text-align: left;
+    white-space: initial;
+  }
+
+  .input-range {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background-color: transparent;
+    margin-left: 7px;
+    margin-right: 7px;
+    min-height: 44px;
+    min-width: 240px;
+    width: 70%;
+  }
+  
+  .input-range:disabled {
+    opacity: 30%;
+  }
+
+  .input-range::-webkit-slider-runnable-track {
+    /* Hides the slider so that custom slider can be made */
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: linear-gradient(to right, var(--terra-slider-track-selected-range, #007cc3) 0%, var(--terra-slider-track-selected-range, #007cc3) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range, #d3d4d5) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range, #d3d4d5) 100%);
+    border-radius: 3px;
+    height: 6px;
+    width: 6px;
+  }
+
+  .input-range::-webkit-slider-thumb {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: var(--terra-slider-thumb-background, #007cc3);
+    border-color: var(--terra-slider-thumb-border-color, #0065a3);
+    border-radius: 50%;
+    border-style: solid;
+    border-width: 1px;
+    box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.25);
+    cursor: pointer;
+    height: 24px;
+    margin-top: -8px;
+    width: 24px;
+  }
+
+  .input-range:focus::-webkit-slider-thumb {
+    border-color: var(--terra-slider-thumb-focus-border-color, #007cc3);
+    border-width: 1px;
+    outline: var(--terra-slider-thumb-focus-outline, 2px dashed #000);
+    outline-offset: 1px;
+  }
+
+  .input-range:hover::-webkit-slider-thumb {
+    background: var(--terra-slider-thumb-hover-background, #0065a3);
+    border-color: var(--terra-slider-thumb-hover-border-color, #0065a3);
```

Comment:
In your logic the background color and border color are the same.  Do you just not need a border?

Reply:
these were same for hover for some reason in the call-outs. I will double check with UX on this.

Reply:
I think some of the newer PRs used "background".  You definitely don't need both styles though.  We don't need to consult with UX on that.  Using background would allow other themes to use a gradient color, if desired.

Reply:
@cm9361 this is probably to ensure the border color is overridden for the hover style. There's several ways this could be accomplished but this seems alright.

Reply:
@eawww You only need one of the styles.  The background style can be set to a color.  We don't want to have two styles that can conflict each other with new code.

Reply:
I have removed the border color wherever they are same as the background color. I checked that there are no visual problems or thumb size changes after removing this.